rpc/cloudagent/network/loadbalancer/moc_cloudagent_loadbalancer.proto

```diff
@@ -39,6 +46,7 @@ message LoadBalancer {
 	Status status = 10;
 	Tags tags = 11;
 	uint32 replicationCount = 12;
+	repeated LoadbalancerInboundNatRule inboundNatRules = 13;
```

Comment:
Why is LoadbalancerInboundNatRule  embedded in LoadBalancingRule? Both share the same attributes like frontEndPort etc

Reply:
it's in LoadBalancer not LoadBalancingRule. looks like github diff is making it a little misleading. If you expand the view it will show up correctly.